master/static/srv/get_task_logs_fields.sql

```diff
@@ -0,0 +1,79 @@
+SELECT
```

Comment:
yeah, i'm 99% sure i _can't_ do `with (select * from task_logs where id=..) my_task_logs ...` because cte's will force optimization fences or something and otherwise i'm not sure how to get the same result cleaner. any ideas?

Reply:
but following up, we really should upgrade postgres, but even if we do, we should be careful